### DISCUSSION TITLE
Welcome to freemarker-parser Discussions!

### DISCUSSION THREAD

**Discussion by armano2** [armano2/freemarker-parser#202] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by timothy-o-johnson** [comment]
Hello! 
I'm having problems getting this to work on my machine. I'd like to use a template from an ftl file. I'm not sure if that's the problem.
Can you create a video?
Will this create a parsed version of the ftl file? 

My end goal is to create a JSON type object that I can run tests against.  Can this help with that?

Many thanks!
